packages/components/src/navigator/navigator-to-parent-button/README.md

```diff
@@ -4,6 +4,8 @@
 This feature is still experimental. “Experimental” means this is an early implementation subject to drastic and breaking changes.
```

Comment:
Sounds like we can remove the experimental callout now.

Reply:
I was planning on removing those experimental callouts and Storybook tags once we also export the component without the experimental prefix (in a follow-up PR)